containers/Connector/Connector.tsx

```diff
@@ -61,9 +61,16 @@ const useConnector = () => {
 
 	useEffect(() => {
 		const init = async () => {
-			// TODO: need to verify we support the network
 			const networkId = await getDefaultNetworkId();
 
+			if (!window.ethereum) {
+				setAppReady(true);
+				setNetwork({ name: NetworkName.Mainnet, id: networkId, useOvm: false });
+				setSynthetixjs(synthetix({ networkId, useOvm: false }));
+				return;
+			}
+
+			// TODO: need to verify we support the network
 			const provider = loadProvider({
 				networkId,
 				//infuraId: process.env.NEXT_PUBLIC_INFURA_PROJECT_ID,
```

Comment:
can be removed, no?

Reply:
Will be removed in this PR: https://github.com/Synthetixio/staking/pull/780
`loadProvider` throws if given a network synthetix doesn't support